pkg/apis/testharness/v1beta1/test_types.go

```diff
@@ -152,6 +152,10 @@ type TestAssert struct {
 	Collectors []*TestCollector `json:"collectors,omitempty"`
 	// Commands is a set of commands to be run as assertions for the current step
 	Commands []TestAssertCommand `json:"commands,omitempty"`
+
+	ResourceRefs []TestResourceRef `json:"resourceRefs,omitempty"`
+
+	AssertExpressions AnyAllExpressions `json:"assertExpressions,omitempty"`
```

Comment:
This does not follow the structure in the KEP: `assert(Any|All).celExpr`. Is it simply the prototype of an old iteration of the KEP?

Reply:
In hind-sight,
```yaml
assert:
  any:
    ...
  all:
    ...
```
would be less verbose. WDYT?

Reply:
I think the verbosity is the nearly same - the number of letters is the same and the KEP version has one colon fewer.
And the KEP version is flatter, less indentation to think about, so I'd be for sticking to the KEP version. 🤷🏻